libs/core/langchain_core/messages/system.py

```diff
@@ -4,8 +4,29 @@
 
 
 class SystemMessage(BaseMessage):
-    """Message for priming AI behavior, usually passed in as the first of a sequence
+    """Message for priming AI behavior.
+
+    The system message is usually passed in as the first of a sequence
     of input messages.
+
+    Example:
+
+        .. code-block:: python
+
+            from langchain_core.messages import HumanMessage, SystemMessage
+
+            messages = [
+                SystemMessage(
+                    content="You are a helpful assistant! Your name is Bob."
```

Comment:
nit don't need to specify content as kwarg

Reply:
(and imo cleaner not to)

Reply:
Most of our examples still do -- I'd prefer doing an upgrade a bit later once more folks are on the newest version of core?